### PR TITLE
Adding my username to the list of maintainers

### PR DESCRIPTION
**Pull request by mayaCostantini**
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Adding my username to the list of maintainers

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ managers:
         - harshad16
         - pacospace
         - sesheta
+        - mayacostantini
```
Comment (codificat) [inline .thoth.yaml]: As this repo has an OWNERS file and the bot can obtain maintainer information from there, the suggestion here would be to remove the `maintainers` section from `.thoth.yaml` alltogether, and instead make sure that the list of `approvers` in the OWNERS file has the proper list:

https://github.com/thoth-station/report-processing/blob/master/OWNERS#L3

**Review by codificat — CHANGES_REQUESTED** [review]
We had a chat about this in a recent tech talk and I believe that the consensus was to use `approvers` in OWNERS instead of `maintainers` in repos where that's possible

File: .thoth.yaml (hunk 1 above)
Comment (mayaCostantini) [reply .thoth.yaml]: It is done, thank you!

**Review by fridex — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**